nextcord/components.py

```diff
@@ -202,7 +208,59 @@ def to_dict(self) -> ButtonComponentPayload:
         return payload  # type: ignore
 
 
-class SelectMenu(Component):
+class BaseSelectMenu(Component):
```

Comment:
This should probably be `SelectMenuBase` as that appears to be the convention for naming Base objects elsewhere in the lib.